### ISSUE TITLE
Support for parameterizable ilivalidator calls

### ISSUE THREAD

**Issue by olivergrimm** [GeoWerkstatt/interlis-testbed-runner#14]
Implementierung der Unterstützung für parametrisierbare ilivalidator calls

(FYI @edigonzales)

**Comment by edigonzales** [comment]
@olivergrimm https://github.com/GeoWerkstatt/interlis-testbed-runner/blob/main/src/main/java/ch/geowerkstatt/interlis/testbed/runner/validation/InterlisValidator.java#L35

Bleibt das eigentlich so? D.h. man muss vorgängig den ilivalidator bei sich installieren anstatt ilivalidator als normale Dependency zu verwenden?

**Comment by beistehen** [comment]
> Unterstützung für parametrisierbare ilivalidator calls

Damit ist also auch die Angabe einer config-Datei gemeint (worin man dann ein Validierungsmodell referenzieren kann)?

**Comment by olivergrimm** [comment]
Der Support für den Aufruf mittels --config wird im Rahmen des Projekts DMAV Testbed umgesetzt

**Comment by olivergrimm** [comment]
> @olivergrimm https://github.com/GeoWerkstatt/interlis-testbed-runner/blob/main/src/main/java/ch/geowerkstatt/interlis/testbed/runner/validation/InterlisValidator.java#L35
> 
> Bleibt das eigentlich so? D.h. man muss vorgängig den ilivalidator bei sich installieren und dann wird er als externes Programm aufgerufen anstatt ilivalidator als normale Dependency zu verwenden?

ja, vorerst sind daran keine Änderungen geplant. Neu soll auch der IGChecker eingebunden werden können. Mit dieser "Architektur" (ohne Dependency) unterstützen wir die Unabhängigkeit der Validierungs-Tools vom Runner
